backstop/tests/tree.html

```diff
@@ -0,0 +1,1305 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Tree | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        width: auto;
```

Comment:
This change makes test images bigger. I would say either remove this or you can add viewport like this `viewports: [{ width: 800, height: 600 }],` (check `expandable-block.js`).

Reply:
I vote adding `viewports: [{ width: 800, height: 600 }],` to the JS.

Reply:
Added